src/components/Footer/Office.js

```diff
@@ -13,7 +13,7 @@ const Office = ({ name, telephone, email, streetAddress }) => (
   <PaddedCol width={[1, 1, 1, 1 / 2, 1 / 2, 1 / 4]}>
     <Subtitle reverse>{name}</Subtitle>
     <Padding top={1} bottom={1}>
-      {streetAddress.map((address, i) => (
+      {streetAddress.map(address => (
         <StyledBodyPrimary key={address}>{address}</StyledBodyPrimary>
```

Comment:
Better to change this to `key={i}`?

Reply:
Using a map index arg as a key isn't a great pattern

https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-array-index-key.md

Reply:
address is unique enough here and its not dynamic so it'll be ok

Reply:
kk